src/ngraph_backend_config.cc

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "ngraph_backend_config.h"
+
+using namespace std;
+
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
+BackendConfig::BackendConfig(string backend_name) {
+  NGRAPH_VLOG(3) << "BackendConfig() ";
+  backend_name_ = backend_name;
+  optional_attributes_ = {"_ngraph_device_config"};
+}
+
+string BackendConfig::join(unordered_map<string, string> optional_parameters) {
+  NGRAPH_VLOG(3) << "JOIN";
```

Comment:
If you add a log message - add more information such as the length of the map may be?